src/compiler/scala/tools/nsc/typechecker/Contexts.scala

```diff
@@ -1108,10 +1108,10 @@ trait Contexts { self: Analyzer =>
       //
       // A type-import-on-demand declaration never causes any other declaration to be shadowed.
       //
-      // Scala: Bindings of different kinds have a precedence deﬁned on them:
+      // Scala: Bindings of different kinds have a precedence defined on them:
```

Comment:
I noticed this recently, and out of curiosity I hope somebody explains how or why it is produced. Also, those files without a terminal newline.